Sources/Kitura/Kitura.swift

```diff
@@ -56,21 +56,25 @@ public class Kitura {
     ///### Usage Example: ###
     ///```swift
     /// let router = Router()
-    /// Kitura.addHTTPServer(onPort: 8080, with: router)
+    /// Kitura.addHTTPServer(onPort: 8080, with: router, address: "localhost")
     ///```
     /// - Parameter onPort: The port to listen on.
     /// - Parameter with: The `ServerDelegate` to use.
+    /// - Parameter address: The address to listen on, for example "localhost". The default is nil, which listens on
+    ///             all addresses.
```

Comment:
I think it makes sense to put the address param after the port, before the delegate (router), so that related values are together.

Also I prefer the name `onAddress` as it matches with `onPort`.